.gitignore

```diff
@@ -1,5 +1,7 @@
 obj/
 bin/
 *.user
-wwwroot/
-node_modules/
\ No newline at end of file
+# Temporarily adding wwwroot to the version control.
+# wwwroot/
+node_modules/
+.idea

```

Comment:
Why did you add these?

Reply:
.idea is Rider's .vs folder that must be ignored, but you are right, it can be on the superrepository-level.